awx/ui/client/src/inventories-hosts/inventories/related/hosts/related/nested-groups/host-nested-groups-list.controller.js

```diff
@@ -48,24 +44,6 @@
 
         }
 
-        function buildStatusIndicators(group){
```

Comment:
for this view:
![Screenshot from 2019-12-05 15-59-37](https://user-images.githubusercontent.com/9753817/70273581-5bb00c80-1778-11ea-80c9-5e6b51f20d3f.png)